devices/stm32h562.yaml

```diff
@@ -28,7 +28,9 @@ _include:
   - ../peripherals/gpio/gpio_with_brr.yaml
   - ../peripherals/gpio/gpio_with_hslvr.yaml
 
+  - common_patches/rcc/h56x_h57x.yaml
   - ../peripherals/rcc/rcc_h5.yaml
```

Comment:
`rcc_h5.yaml` is already included in `peripherals/rcc/rcc_h56x_h57x.yaml`. Remove this line, please. Same for h56x/h57x

Reply:
Do you want me to revert and reopen this PR or submit a new PR?

Reply:
New PR, of cause.